sequencer/onchain.go

```diff
@@ -122,7 +121,8 @@ func (s *Sequencer) processTransitionOnChain() {
 	})
 }
 
-func (s *Sequencer) pushTransitionToContract(processID []byte,
+func (s *Sequencer) pushTransitionToContract(
+	processID []byte,
```

Comment:
Use types.HexBytes here, then you don't need to make the Hex conversion, just call processID.String()